build.sbt

```diff
@@ -110,7 +110,7 @@ val compileDependencies = Seq(
   "uk.gov.hmrc" %% "logback-json-logger" % "5.1.0",
   "uk.gov.hmrc" %% "play-language" % "5.0.0-play-28",
   "org.webjars.npm" % "accessible-autocomplete" % "2.0.3",
-  "uk.gov.hmrc" %% "play-frontend-hmrc" % "0.93.0-play-28"
+  "uk.gov.hmrc" %% "play-frontend-hmrc" % "3.0.0-play-28"
```

Comment:
Please upgrade to version `3.2.0-play-28` as it's the latest version today.

Reply:
Play-frontend-hmrc is now updated to 3.2.0.